src/content/docs/service-level-management/create-slm.mdx

```diff
@@ -332,9 +334,11 @@ You can create SLIs and SLOs from several places on [in our UI](https://one.newr
 
 * Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Service levels**. You can associate the SLI with any entity across your accounts, including workloads.
 * From the **Service levels** page in any APM Service, Browser application, or Synthetic monitor. The SLI will be associated with that specific entity.  If you use this starting point, New Relic will automatically create the most common service level indicators for this entity type, based on the latest available data.
-* From the **Service levels** tab in any workload. You can associate the SLI with any entity in the workload, or the workload itself.
+* From the **Service levels** tab in any workload. You can associate the SLI with any entity in the workload, or the whole workload. 
+
+Remember that service levels can only be associated with a single account. For details on that, see [the requirements](#requirements).
 
-Follow these steps:
+To create service levels, follow these steps:
```

Comment:
@zuluecho9 I found it a bit confusing since I didn't see any numbers for the steps. What do you think about inserting "Step X" in the collapser titles?

Reply:
it's a great idea, but i was just making a quick edit to this doc about a specific thing for now.